src/adapter/pg/read/target.go

```diff
@@ -0,0 +1,50 @@
+package read
+
+import (
+	"database/sql"
+	"github.com/liampm/static-code-analysis-tool/domain"
+	"github.com/satori/go.uuid"
+)
+
+type PostgresTargetReadRepo struct {
+	db *sql.DB
+}
+
+func TargetRepo(db *sql.DB) *PostgresTargetReadRepo {
+	return &PostgresTargetReadRepo{db: db}
+}
+
+func (repo *PostgresTargetReadRepo) Find(id uuid.UUID) (domain.Target, error) {
+	target := domain.Target{}
+
+	row := repo.db.QueryRow("SELECT * FROM target WHERE id = $1", id)
+
+	// Populate the entity with the information from the row
+	err := row.Scan(&target.Id, &target.ProjectId &target.Name)
+
+	if err == sql.ErrNoRows {
+		return target, err // Return the error so that it can be dealt with
```

Comment:
Again - I've done the same. But if we had more time I'd like to implement our own error rather than using the SQL ones